ravindu14/SliitSpmProject
# Feature/task 502 validate student registration form

How to Test 

1. go to the branch -> feature/task-502-validate-student-registration-form
2. go to the student registration
3.check for the validations
